frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install-wizard.tsx

```diff
@@ -114,18 +114,20 @@ const CreateStorageClusterWizard: React.FC<CreateStorageClusterWizardProps> = ({
   mode,
   lsoNs,
   navUtils,
+  hasNoProvSC,
 }) => {
   const { t } = useTranslation();
   const { appName, ns } = match.params;
   const [state, dispatch] = React.useReducer(reducer, initialState);
   const [showInfoAlert, setShowInfoAlert] = React.useState(true);
   const [inProgress, setInProgress] = React.useState(false);
   const [errorMessage, setErrorMessage] = React.useState('');
+  const [isForceNavigated, setForceNavigated] = React.useState(false);
```

Comment:
Why do we need to maintain two states? May be I am missing something but I am thinking `noProv` would  be suffice.

Reply:
The page keeps refreshing when I only worked with `noProv`, and there was a need to record whether a page has already rendered or not in order to stop that behavior, hence this variable.